benchmark.py

```diff
@@ -265,12 +279,28 @@ def _step():
             for _ in range(self.num_warm_iter):
                 _step()
 
+            if self.cuda_graph:
```

Comment:
Can you add some comments simply mentioning we need to sync the stream, do warm up iterations, then capture the graph.